src/test/java/mops/businesslogic/DirectoryServiceTest.java

```diff
@@ -0,0 +1,284 @@
+package mops.businesslogic;
+
+import mops.SpringTestContext;
+import mops.businesslogic.exception.DeleteAccessPermissionException;
+import mops.businesslogic.exception.ReadAccessPermissionException;
+import mops.businesslogic.exception.WriteAccessPermissionException;
+import mops.exception.MopsException;
+import mops.persistence.DirectoryPermissionsRepository;
+import mops.persistence.FileRepository;
+import mops.persistence.directory.Directory;
+import mops.persistence.permission.DirectoryPermissionEntry;
+import mops.persistence.permission.DirectoryPermissions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.util.List;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.BDDMockito.given;
+
+@SpringTestContext
+@SpringBootTest
+public class DirectoryServiceTest {
+    public static final String ADMINISTRATOR = "administrator";
+    public static final String STUDENTIN = "studentin";
+    public static final String READER = "reader";
+    public static final String INTRUDER = "intruder";
+    public static final String USER = "user";
```

Comment:
```suggestion
    public static final String ADMINISTRATOR = "administrator";
    public static final String STUDENTIN = "studentin";
    public static final String READER = "reader";
    public static final String INTRUDER = "intruder";
    public static final String USER = "user";
```
Maybe put into own "DirectoryServiceTestConfig" or so.

Reply:
I think they are part of the logical and not of the configuration.